src/main/kotlin/no/nav/syfo/application/api/ApiModule.kt

```diff
@@ -73,6 +74,11 @@ fun Application.apiModule(
         aadTokenEndpoint = environment.aadTokenEndpoint,
         redisStore = cache,
     )
+    val tokendingsClient = TokendingsClient(
+        tokenxClientId = environment.tokenxClientId,
+        tokendingsEndpoint = environment.tokenxEndpoint,
```

Comment:
Nå blander vi `tokendingsEndpoint` og `tokenxEndpoint`, kan de hete det samme?

Reply:
tokenxEndpoint kommer fra env-variabelen som nais setter opp for oss når tokenx er enabled. De har vel valgt det navnet siden det referer til et token exchange-endepunkt som ikke nødvendigvis trenger å være tokendings. Men den klienten vi lager "vet vi" skal være mot tokendings, så i den konteksten synes jeg det er mer naturlig å kalle det tokendingsEndpoint.

Reply:
Hvorfor kaller vi det ikke da `tokendingsClientId`, og `tokendingsPrivatJWK`?

Reply:
Det er sant, skal forsøke å gjøre det mer konsistent.

Reply:
Fikset litt nå - gikk tilbake til det første forslaget ditt og har renavnet parameteren til tokenxEndpoint.